File: examples/modulith-clinical-tool-jpa/src/test/java/io/zenwave360/example/clinicaltool/modules/surveys/infrastructure/jpa/inmemory/QuestionRepositoryInMemory.java
package io.zenwave360.example.clinicaltool.modules.surveys.infrastructure.jpa.inmemory;

import io.zenwave360.example.clinicaltool.modules.surveys.core.domain.*;
import io.zenwave360.example.clinicaltool.modules.surveys.core.outbound.jpa.QuestionRepository;
import java.math.*;
import java.time.*;
import java.util.*;

public class QuestionRepositoryInMemory extends InMemoryJpaRepository<Question> implements QuestionRepository {
}

File: examples/modulith-klinical-tool-jpa/src/test/kotlin/io/zenwave360/example/clinicaltool/modules/surveys/repository/jpa/inmemory/QuestionRepositoryInMemory.kt
package io.zenwave360.example.clinicaltool.modules.surveys.repository.jpa.inmemory

import io.zenwave360.example.clinicaltool.modules.surveys.domain.*
import io.zenwave360.example.clinicaltool.modules.surveys.repository.jpa.QuestionRepository
import java.math.*
import java.time.*
import java.util.*

class QuestionRepositoryInMemory : InMemoryJpaRepository<Question, Long>(), QuestionRepository {}
